workers/docker-worker/src/lib/util/artifact_download.js

```diff
@@ -38,13 +38,22 @@ module.exports = async function(queue, stream, taskId, artifactPath, destination
     try {
       let expectedSize = 0;
       let receivedSize;
-      let req = request.get(artifactUrl);
+      let req = request.get({
+        method: 'GET',
+        url: artifactUrl,
+        gzip: true,
```

Comment:
I wasn't able to find the `gzip` option in the `superAgent` docs or (latest) source code. Is it necessary?

Reply:
So this is pretty crazy, but docker-worker is full of crazy.  In this case we're using [`request`](https://classic.yarnpkg.com/en/package/request), not superagent.  (Really I suspect this was just written this way long before `request` was deprecated).  Here are the docs, such as they are -- https://github.com/request/request#examples

Reply:
Oops, I've been staring at a lot of `const request = require('superagent');`, I missed this was different. I guess a lot of code was using `request`, and that's why it isn't `const superagent = require('super agent')`

Reply:
Believe it or not, superagent itself recommends
```
const request = require('superagent');
```
I have no idea why..